src/index.ts

```diff
@@ -4,22 +4,21 @@ dotenv.config();
 import { setupApp } from './app';
 import { close } from './services/client';
 import { initiateLogger } from './services/logger';
+import { Server } from 'http';
```

Comment:
Forgot this one nitpick! We should probably have libraries imported above relative imports

Reply:
good catch!